examples/blocked_cholesky_gpu.py

```diff
@@ -175,7 +176,8 @@ async def test_blocked_cholesky():
 
         # Copy and layout input
         a1 = a.copy()
-        ap = a1.reshape(n // block_size, block_size, n // block_size, block_size).swapaxes(1,2)
+        mapper = LDeviceGridBlocked(n // block_size, n // block_size, devices=get_current_devices())
```

Comment:
This is the trick that creates views - partitions on the current device are created as views.